src/CardContextComponent/CardsContextProvider.js

```diff
@@ -1,36 +1,42 @@
 import React, { useState } from 'react';
+import {getCardList} from '../api/cards.api';
 
 
 const CardsContext = React.createContext({});
 
 
-function CardsContextProvider(Children, { initCardList }) {
+function CardsContextProvider(Children) {
 
   return function Child(props) {
 
-    const [cardsContext, setCardsContext] = useState(() => ({
-      cards: initCardList,
-      setCards: cards => setCardsContext(state => ({
-        ...state,
-        cards
-      })),
-      addCard: card => setCardsContext(state => ({
-        ...state,
-        cards: [card, ...state.cards]
-      })),
-      removeSelectedCards: () => setCardsContext(state => ({
-        ...state,
-        cards: state.cards.filter(card => !card.selected)
-      })),
-      editCard: newCard => setCardsContext(state => ({
-        ...state,
-        cards: state.cards.map(
-          one => one.id === newCard.id
-            ? {...one, ...newCard}
-            : one
-        )
-      }))
-    }));
+    const [cardsContext, setCardsContext] = useState(() => {
+      // Fetch cards from bd
+      getCardList().then(cards => cardsContext.setCards(cards));
+
+      return {
+        cards: [],
```

Comment:
I think these lines are really strange. Can you please move getting list move to useEffect for example?
fix it and merge please.